cli/ballerina-cli/src/main/resources/create_cmd_templates/lib/lib.bal

```diff
@@ -1,5 +1,30 @@
-import ballerina/io;
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
 
-public function hello() {
-    io:println("Hello World!");
+# Prints `Hello` with input string name.
+# + name - the input sting name
+# + return - "Hello, " with the input string name
+
+public function hello(string name) returns string {
+    if (name is string) {
```

Comment:
This check is redundant since we already specify the parameter type as `string`. We can remove this condition